modules/consentManagement.js

```diff
@@ -190,7 +332,15 @@ function lookupIabConsent(cmpSuccess, cmpError, hookConfig) {
  * @param {object} reqBidsConfigObj required; This is the same param that's used in pbjs.requestBids.
  * @param {function} fn required; The next function in the chain, used by hook.js
  */
-export function requestBidsHook(fn, reqBidsConfigObj) {
+export function requestBidsHook(fn, reqBidsConfigObj, isCCPA = false) {
+  let userModule = userCMP;
+  let processFn = processCmpData;
+
+  if (isCCPA) {
+    userModule = 'ccpa';
```

Comment:
what's this?

Reply:
A workaround for the global params. I could probably refactor it away.